Dungeon_of_Munchkinomicon.py

```diff
@@ -20,19 +20,28 @@ def death(name, reason):
 	print "You died because %s. What did you think would happen?" % reason
 	redo(name)
 	
+def getConfirmation(ask):
+        print ask
+        choice = raw_input(prompt).lower()
+        print choice
+        if "yes" in choice :
+                return True
+        else:
+                return False
+
+
 def redo(name):
 	global fairy
 	global katana
-	print "Try again, %s?" % name
-	tryagain = raw_input(prompt)
-	if "yes" in tryagain or "Yes" in tryagain:
+	tryAgain = getConfirmation("Try again, %s?" % name)
+	if tryAgain:
 		if name == "Cthulu":
-			username
-		else:

```

Comment:
I don't think you intended an else statement here. I ran into a bug where setting my name to Cuthulu would immediately finish the game, ask me to replay or not, and if I said yes it would ask for a new name BUT then not start a new game because of this else statement.